lib/es5/index.js

```diff
@@ -390,6 +390,11 @@ Parser.prototype.__push = function (line) {
   if (this.listenerCount('record')) {
     this.emit('record', record);
   }
+  // When to is reached set ingore any future calls
+  if (this.count >= this.options.to) {
+    this.isEnded = true;
+    return this.push(null);
+  }
```

Comment:
Moved this block to after the push/emit to ensure the record being processed is returned.